cypress/fixtures/fireboltCalls/account.js

```diff
@@ -0,0 +1,109 @@
+const errorContent = require('../objects/errorContentObjects.js');
+exports.ACCOUNT_VARIABLES = {
+  ACCOUNT: {
+    id: {
+      data: [
+        {
+          type: 'fixture',
+          validations: [
+            {
+              mode: 'deviceContentValidation',
+              type: extractDeviceData('ACCOUNTID'),
+              description: 'Validation of the Account Id',
+            },
+          ],
+        },
+      ],
+    },
+    uid: {
+      data: [
+        {
+          type: 'regEx',
+          validations: [
+            {
+              mode: 'regex',
+              type: '/^(?:.*d)(?:.*[a-zA-Z]).{2,}$/',
+              description: 'Validation of the Account uid',
+            },
+          ],
+        },
+      ],
+    },
+  },
+};
+
+exports.ACCOUNT_ID_UID = {
+  method: resolveAtRuntime('account.{{attribute}}'),
+  params: null,
+  validationJsonPath: 'result',
+  content: resolveAtRuntime('DYNAMIC_FB_CALL_VARIABLES.DEFAULTS.ACCOUNT.{{attribute}}'),
+};
+
+exports.FETCH_ACCOUNT_ID_WHEN_DEVICE_IS_NOT_PROVISIONED = {
+  method: 'account.id',
+  params: {},
+  expected: 'error',
+};
+
+exports.FETCH_ACCOUNT_UID_WHEN_DEVICE_IS_NOT_PROVISIONED = {
+  method: 'account.uid',
+  params: {},
+  expected: 'error',
+};
+
+exports.INVALID_PARAMETER_ERROR_FOR_ACCOUNT_ID = {
+  method: 'account.id',
+  validationJsonPath: 'result',
+  content: errorContent.INVALID_TYPE_PARAMS,
+  expectingError: true,
+};
+
+exports.INVALID_PARAMETER_ERROR_FOR_ACCOUNT_UID = {
+  method: 'account.uid',
+  validationJsonPath: 'result',
+  content: errorContent.INVALID_TYPE_PARAMS,
+  expectingError: true,
+};
+
+exports.PUSH_SESSION_TOKEN = {
+  method: 'manage_account.session',
+  params: { token: 'RmlyZWJvbHQgTWFuYWdlIFNESyBSb2NrcyEhIQ==', expiresIn: 84000 },
```

Comment:
Two comments please:
1. Is this a valid and possibly sensitive value?  (Other hairy numbers below this line are described as "invalid.)"
2. All code files should have headers.  (Includes this file but also others in the pull req.)
Thank you,
Martin Hughes, RDK CMF Compliance Team

Reply:
I've updated the token value to a dummy encoded string.